src/Tasks/IdentifyFeatures.js

```diff
@@ -62,6 +65,61 @@ export var IdentifyFeatures = Identify.extend({
         callback.call(context, undefined, featureCollection, response);
       }
     });
+  },
+  _setGeometry: function (geometry) {
```

Comment:
lets DRY this up and move  `_setGeometry()`   into `L.esri.Util` so that we can reference the same function from both `Query` _and_ `identifyFeatures`